app/src/forms/form/exportService.js

```diff
@@ -133,7 +133,29 @@ const service = {
     };
   },
 
+  _readLatestFormSchema: async (formId) => {
+    return FormVersion.query()
+      .select('schema')
+      .withGraphFetched('submissions')
+      .where('formId', formId)
+      .modify('orderVersionDescending')
+      .first().then((row) => row.schema);
```

Comment:
The then function should probably be on a separate line for readability.
You can probably safely remove the async from this function as it is not awaiting anywhere.